packages/vue_frontend_rpc/test/functional/editor/execute-task.spec.js

```diff
@@ -1,180 +0,0 @@
-import { expect } from "chai";
```

Comment:
why test files from the "editor" folder were deleted? or they were converted to `Editor.spec.js`?